config/routes.rb

```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
+  resources :events
+  resources :users
+  resources :sessions, only: %i[new create destroy]
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root 'events#index'
+
+  get 'signup', to: 'users#new', as: 'signup'
+  get 'login', to: 'sessions#new', as: 'login'
+  get 'logout', to: 'sessions#destroy', as: 'logout'
```

Comment:
### remember we required you to use `resources` try the following, check out the following suggestion :wink: 
```ruby
scope(path_names: { new: 'login', destroy: 'logout' }) do
  resources :session, path: ' '
end
```
>same goes for the `UserController`